### DISCUSSION TITLE
curl failed to verify the legitimacy of the server and therefore could not establish a secure connection to it.

### DISCUSSION THREAD

**Discussion by lzytim** [Homebrew/discussions#1882] [Getting started]
Hey there. I would like to apologize if this is a repeated question. I encountered this error message while attempting to install homebrew. How do I solve this issue to get Homebrew installed? I am currently using a Macbook with an M1 chip. Thank you for your help!

curl: (60) SSL certificate problem: self signed certificate in certificate chain
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.

**Comment by gromgit** [comment]
Sounds like you're using a local proxy with a self-signed cert, or you're being [MITM'd](https://www.vaticannews.va/en/pope/news/2021-01/pope-establishes-world-day-for-grandparents-and-the-elderly.html). Either way, that's not a good thing. :grin: 

If this is "self-inflicted", simply disable your proxy settings and try the install again. If this was a complete surprise, you may want to ask for local IT help.

**Reply by lzytim** [reply]
Yes you are right. My antivirus software had a hand in preventing me to download Homebrew. I disabled my antivirus and the installation proceeded without a hitch. 

Thanks for your help!

**Reply by N-bop** [reply]
I found I had to install xcode and disable my antivirus to download Homebrew, so this was good to know!

**Comment by ooriblestsmshemir** [comment]
Another scenario could be that a known/planned man-in-the-middle is between your computer and the network resource.  A system like Zscaler breaks the SSL/TLS cert, inspects the contents, and signs the page with their own cert.  

A solution would be to use the "-k" or  "--insecure" options to disable certificate verification.  I don't like it, and it's understandable that companies would like to crack and inspect the encrypted infil/exfil traffic.  

I just tested curl -k behind Zscaler and it work to retrieve https://www.isaca.org/-/media/files/isacadp/project/isaca/resources/state-of-cybersecurity-2022_whpsc22_res_eng_0322.pdf
